bom/pom.xml

```diff
@@ -216,18 +216,52 @@
 
       <dependency>
         <groupId>com.redhat.service.smartevents</groupId>
-        <artifactId>manager-integration-tests</artifactId>
+        <artifactId>manager-integration-tests-common</artifactId>
         <version>${project.version}</version>
       </dependency>
       <dependency>
         <groupId>com.redhat.service.smartevents</groupId>
-        <artifactId>manager-integration-tests</artifactId>
+        <artifactId>manager-integration-tests-common</artifactId>
         <version>${project.version}</version>
         <classifier>sources</classifier>
       </dependency>
       <dependency>
         <groupId>com.redhat.service.smartevents</groupId>
-        <artifactId>manager-integration-tests</artifactId>
+        <artifactId>manager-integration-tests-common</artifactId>
+        <version>${project.version}</version>
+        <classifier>javadoc</classifier>
+      </dependency>
+      <dependency>
+        <groupId>com.redhat.service.smartevents</groupId>
+        <artifactId>manager-integration-tests-v1</artifactId>
```

Comment:
Do we need the BOM declarations anywhere?

Reply:
Technically, we need only `manager-intgration-tests-common`. However, there were the original `manager-integration-tests`, so I kept it.